src/components/Shared/UI/DTESelect/DTESelect.tsx

```diff
@@ -28,6 +28,21 @@ const StyledSelect = styled(Select)`
     }
   }
 `;
+
+function capitalizeWords(input: string) {
+  const lowercasedInput = input.toLowerCase();
```

Comment:
no need to introduce an additional variable, just use chaining as you have in the other methods